packages/compass-data-modeling/src/components/diagram-editor.tsx

```diff
@@ -423,6 +570,11 @@ const ConnectedDiagramContent = connect(
     onFieldSelect: selectField,
     onDiagramBackgroundClicked: selectBackground,
     onCreateNewRelationship: createNewRelationship,
+    onDeleteCollection: deleteCollection,
+    onDeleteRelationship: deleteRelationship,
+    onDeleteField: removeField,
+    onUndoClick: undoEdit,
+    onRedoClick: redoEdit,
```

Comment:
For some cases having the hotkeys defined makes sense, but I think for some of them you can keep them closer to the UI that they are "attached" to, for example instead of adding undo / redo here, you can use `useHotkey` hook to define these hotkeys near the button elements that do undo / redo